src/lib/tokenPrices.ts

```
@@ -14,15 +14,15 @@ export const tokenToUsd = async (token: string): Promise<number> => {
 
 	if (token === 'LOOT') {
 		priceInUsd = await getLootPrice();
+		if (!priceInUsd) return 0;
```

This function should throw an exception if it fails, not zero